tests/context.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+
+import sys
+import os
+sys.path.insert(0, os.path.abspath('..'))
```

Comment:
This is just a note to remind me to ask you about this when we do a walk through. I haven't seen this before and want to learn more.